### DISCUSSION TITLE
content alignment in qselect

### DISCUSSION THREAD

**Discussion by albukzek** [quasarframework/quasar#13795] [General - Components / Directives / etc]
Hello. Unable to align the context in the select. How can I do that? When inspecting, it can be seen that the content area is higher than the select border
![image](https://user-images.githubusercontent.com/31003788/175873193-9ea5682c-5390-40bf-9899-a48fba3dfc6e.png)

**Comment by MilosPaunovic** [comment]
Please provide a Codepen reproduction so the community can provide better support.
